docs/DeveloperGuide.adoc

```diff
@@ -279,87 +279,362 @@ Refer to the guide <<DevOps#, here>>.
 
-_{More to be added}_
+_{All user stories can be viewed in our issue tracker.}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is `Volant (V)` and the *Actor* is the `user`, unless specified otherwise)
+
+[discrete]
+=== UC01 - Adding New Trip
+
+**Precondition: **Start from home page
+
+*Guarantee:* A new Trip will be created
+
+*MSS*
+
+1. User enters details of trip to be added
+2. V writes added trip into data files and displays new trip in the trip list
++
+Use case ends
+
+*Extensions*
+
+[none]
+* 1a. V detects invalid characters in trip details
+* 1a1. V outputs error informing user of invalid characters
+* 1a2. V terminates the command
++
+Use case ends
+
+[none]
+* 1b. V detects missing fields in trip details
+* 1b1. V outputs error informing user of missing fields
+* 1b2. V terminates the command
++
+Use case ends
+
+[none]
+* 1c. V is unable to detect a valid keyword
+* 1c1. V outputs error informing user of invalid keyword
+* 1c2. V terminates the command
++
+Use case ends
+
+[discrete]
+=== UC02 - Editing Existing Trip
+
+**Precondition: **User is in the home page
+
+**Guarantee: **Information of existing trip will be changed
+
+*MSS*
+
+1. User requests to edit trip at specified index
+2. V requests for details to be overwritten/edited
+3. User enters new trip details
+4. V writes new details back into data files and displays edited trip in the trip list
++
+Use case ends
+
+*Extensions*
+
+[none]
+* 3a. V detects an invalid index i.e. index out of range or negative index
+* 3a1. V outputs error informing user of invalid index specified
+* 3a2. V terminates the command
++
+Use case ends
+
+[none]
+* 3b. V is unable to detect a valid keyword
+* 3b1. V outputs error informing user of invalid keyword
+* 3b2. V terminates the command
++
+Use case ends
+
+[discrete]
+=== UC03 - Accessing a Trip Folder
+
+*Precondition:* User is in the hope page and the trip to access exists
+
+*Guarantee:* User is now in Trip Folder
+
+*MSS*
+
+1. User changes directory to specific trip folder in trip list using index
+2. V loads data in trip folder and displays items in trip folder
++
+Use case ends
+
+*Extensions*
+
+[none]
+* 1a. V detects an invalid index i.e. index out of range or negative index
+* 1a1. V outputs error informing user of invalid index specified
+* 1a2. V terminates the command
++
+Use case ends
+
+[none]
+* 1b. V is unable to detect a valid keyword
+* 1b1. V outputs error informing user of invalid keyword
+* 1b2. V terminates the command
++
+Use case ends
+
+[discrete]
+=== UC04 - Accessing Pages (Contact List / Trip Itinerary / Journal)
+
+*Precondition:* User is in the home page
+
+*Guarantee:* User will be moved to the desired page
+
+*MSS*
+
+1. User navigates to a specific trip (UC03)
+2. User requests to navigate to the desired page in current trip
```

Comment:
hi! to add on, as we should alternate `User` and `V`, instead of having 2 consecutive  `User` (_rationale: use-case diagram shows user-system interaction_)